### ISSUE TITLE
Make the package self-contained

### ISSUE THREAD

**Issue by mnaoumov** [clariuslabs/TransformOnBuild#14]
Can you move all dependencies inside the package, i.e. TextTransform.exe and all its dependencies. It would be great if you can tranform T4 on a build server without any parts of Visual Studio installed. Do you see any Microsoft license issues with that?

**Comment by clariusconsulting** [comment]
Yes, licensing is an issue. I'm pretty sure you can't redistribute it.

Alternatively, maybe we can bundle the Mono port??

**Comment by mnaoumov** [comment]
@clariusconsulting Unfortunately Mono port is too raw at the moment. I tried it today and it failed in a very simple template. I will send pull requests to Mono to fix the issues I found soon but I don't think it is ready to replace Microsoft's implementation yet

**Comment by mnaoumov** [comment]
Raised an issue for Microsoft https://visualstudio.uservoice.com/forums/121579-visual-studio-2015/suggestions/12508017-create-an-official-nuget-package-with-texttransfor
